Sources/MapboxCoreNavigation/EHorizon/RoadGraph.swift

```diff
@@ -26,7 +26,7 @@ public final class RoadGraph {
      
      - returns: A line string corresponding to the given edge identifier, or `nil` if the edge is inaccessible.
      */
-    public func edgeShape(edgeIdentifier: ElectronicHorizon.Edge.Identifier) -> LineString? {
+    public func edgeShape(edgeIdentifier: RoadGraph.Edge.Identifier) -> LineString? {
```

Comment:
Seems that we don't need `RoadGraph.` prefix here and in the `edgeMetadata` method.